ruby/lib/trailer_vote/api/issue/create.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require 'trailer_vote/api/issue'
+
+module TrailerVote
+  module Api
+    class Issue
```

Comment:
You can create an `Issue` directly from the "Issue" (`ruby/lib/trailer_vote/api/issue.rb`) api:

```suggestion
    class Issue
      
       def create
         Create.new(configuration: configuration)
       end

```

The `configuration` comes from the instance accessor on the "Issue" api.